### DISCUSSION TITLE
На YouTube не загружаются видео с моб. интернета, 2-х операторов подскажите конфиг для ByeDPI

### DISCUSSION THREAD

**Discussion by BestCoder009** [dovecoteescapee/ByeDPIAndroid#209] [Q&A]
Запускал видео на youtube с 2 сим карт со сбермобайла и с МТС с приложением ByeDPI аватарки на каналах на сайте грузят , видео понемногу иногда загружает на минимальном качестве и видео, очень медленно модель телефона техно спарк 10, в ByeDPI настройки менял все равно не грузит скрин настроек прилагается. Какой конфиг, поставить или как иначе скорость сделать нормальной подскажите пожалуйста. П. С. я не силён в программировании (( 🙂
![Screenshot_20241221-112050](https://github.com/user-attachments/assets/554c1af8-32a2-4180-b399-2138edefa6e7)

**Comment by lolyinseo** [comment]
https://github.com/romanvht/ByeDPIAndroid
